packaging/installer/README.md

```diff
@@ -347,13 +347,14 @@ cd netdata
 
 To install Netdata on pfSense, run the following commands (within a shell or under the **Diagnostics/Command** prompt within the pfSense web interface).
 
-Note that the first four packages are downloaded from the pfSense repository for maintaining compatibility with pfSense, Netdata and Python are downloaded from the FreeBSD repository.
+Note that the first four packages are downloaded from the pfSense repository for maintaining compatibility with pfSense, Netdata, Judy and Python are downloaded from the FreeBSD repository.
 
 ```sh
 pkg install pkgconf
 pkg install bash
 pkg install e2fsprogs-libuuid
 pkg install libuv
+pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/Judy-1.0.5_2.txz
 pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/python36-3.6.9.txz
 pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/netdata-1.15.0.txz
```

Comment:
Should the version number here be updated to 17.1?

Reply:
Case we bring 1.17.1 we will need to force our users to create a manual link, because the current package is linked with a difference version and Netdata does not start, but I agree that this would be the correct. Should we ask user to create the link for the library?

Reply:
These are moving targets so expect breakage quite frequently if you're going to hardlink packages. There's a quartely branch but using the provided package framework is the best way to avoid surprises.

Reply:
```
# pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/netdata-1.15.0.txz
pkg: http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/netdata-1.15.0.txz: Not Found
```

The only package available in the `latest` repository is `netdata-1.17.1.txz`

Reply:
A very good point Vladimir with this last I am convinced that it is necessary to add the `ln` command that I was avoiding until now, but I think we need to fix this for our users only install packages.

Reply:
After adding the Judy package the following works:
```
pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/netdata-1.17.1.txz
```
Please update the PR with the change to the version number.

Reply:
Nice, I already updated with the last commit.